services/shared/src/main/java/com/wedriveu/services/shared/rabbitmq/nearest/VehicleResponse.java

```diff
@@ -0,0 +1,79 @@
+package com.wedriveu.services.shared.rabbitmq.nearest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static com.wedriveu.services.shared.utilities.Constants.*;
+
+/**
+ * Created by Marco on 04/08/2017.
+ */
+public class VehicleResponse {
+
+    @JsonProperty(CAR_LICENCE_PLATE)
```

Comment:
I messed with root shared module bringing VehicleResponse there. 
I found out there is no way for the inner modules to read outer modules classes.